src/test/java/emissary/core/IBaseDataObjectHelperTest.java

```diff
@@ -353,4 +356,39 @@ void testAddParentInformationToChildren() {
 
         assertTrue(true);
     }
+
+    @Test
+    void testGetPreferredData() {
```

Comment:
Test name (`testGetPreferredData`) doesn't match the method-under-test (`findPreferredDataByRegex`)